Sources/TidalAPI/Utils/APIError.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+public struct APIError: LocalizedError, Equatable {
```

Comment:
Now that you renamed the package from `API` to `TidalAPI`, this should be `TidalAPIError`?

Reply:
Good point!